charts/kueue/templates/rbac/auth_proxy_role.yaml

```diff
@@ -1,17 +1,17 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: {{ include "kueue.fullname" . }}-proxy-role
+  name: '{{ include "kueue.fullname" . }}-proxy-role'
 rules:
-- apiGroups:
-  - authentication.k8s.io
-  resources:
-  - tokenreviews
-  verbs:
-  - create
-- apiGroups:
-  - authorization.k8s.io
-  resources:
-  - subjectaccessreviews
-  verbs:
-  - create
+  - apiGroups:
+      - authentication.k8s.io
```

Comment:
There is an [issue](https://github.com/mikefarah/yq/issues/825) regarding indents which **yq** makes. The flag -I0 doesn't work properly, that's mean ignore indent. Once the problem problem is solved, I will return the indents back.